### PR TITLE
build!: use setuptools-scm to declare version

### PR DESCRIPTION
**Pull request by jsstevenson**

### CODE REVIEW

File: src/therapy/__init__.py [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
+from importlib.metadata import PackageNotFoundError, version
 from pathlib import Path
 
-from .version import __version__  # noqa: F401
-
 APP_ROOT: Path = Path(__file__).resolve().parents[0]
 
 
+try:
+    __version__ = version("{{ cookiecutter.project_slug }}")
```
Comment (korikuzma) [inline src/therapy/__init__.py:10]:
```suggestion
    __version__ = version("thera-py")
```

**Review by korikuzma — CHANGES_REQUESTED** [review]

File: src/therapy/__init__.py (hunk 1 above)
Comment (jsstevenson) [reply src/therapy/__init__.py:10]: f!

File: tests/unit/test_query.py [hunk 2]
```diff
@@ -756,14 +756,14 @@ def test_service_meta(search_handler, normalize_handler):
     response = search_handler.search(query)
     service_meta = response.service_meta_
     assert service_meta.name == "thera-py"
-    assert service_meta.version >= "0.2.13"
+    assert service_meta.version != "unknown"
```
Comment (korikuzma) [inline tests/unit/test_query.py:759]: Ah, I should have done this for disease or gene? I forget which one made this check

**Review by korikuzma — APPROVED** [review]

File: tests/unit/test_query.py (hunk 2 above)
Comment (jsstevenson) [reply tests/unit/test_query.py:759]: eh I don't really think it matters but at some point we can just run this one around everywhere

**Pull request merged**